pyproject.toml

```diff
@@ -10,8 +10,8 @@ repository = "https://github.com/pureskillgg/dsdk"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-boto3 = "^1.12.32"
-pandas = "^1.2.1"
+boto3 = "^1.34"
+pandas = "1.5.3"
```

Comment:
This should be `^1.5.3"` @billfreeman44

Reply:
@razor-x This is the last version before the 2.0.0 version, which i want to move to. The migration guide said to get to that specific version, then test that everything works with no warnings, then we should be good to go. This is specific, but later it'll be "^2.something.something"